packages/pn-pa-webapp/src/components/Notifications/FilterNotifications.tsx

```diff
@@ -94,8 +133,8 @@ const FilterNotifications = forwardRef(({ showFilters }: Props, ref) => {
     /** onSubmit populates filters */
     onSubmit: (values) => {
       const currentFilters = {
-        startDate: values.startDate,
-        endDate: values.endDate,
+        startDate: values.startDate ?? sixMonthsAgo,
```

Comment:
There is something that doesn't add up to me. Before we didn't need this but now yes, why?